### PR TITLE
Tutorial/deployment

### PR DESCRIPTION
**Pull request by brentguf**
Mainly some updates to reflect Netlify's current UI.

### CODE REVIEW

File: TUTORIAL.md [hunk 1]
```diff
@@ -2408,7 +2408,7 @@ And scroll down to **Heroku Postgres**:
 
 <img alt="Screen Shot 2020-02-03 at 3 23 48 PM" src="https://user-images.githubusercontent.com/300/73703883-556ddc00-46a6-11ea-8777-ee27d2202e0e.png">
 
-Click that and then on the detail page that comes up click the **Install Heroku Postgres** button that top right. On the next screen tell it you want to connect it to the app you just created, then click **Provision add-on**:
+Click that and then on the detail page that comes up, click the **Install Heroku Postgres** button that's at the top right. On the next screen tell it you want to connect it to the app you just created, then click **Submit Order Form**:
 
 <img alt="Screen Shot 2020-02-03 at 3 24 15 PM" src="https://user-images.githubusercontent.com/300/73703930-64548e80-46a6-11ea-9f1b-e06a183834f4.png">
```
Comment (brentguf) [inline TUTORIAL.md:2413]: This screenshot also needs to be updated with:

heroku-submit-add-on

Comment (Tobbe) [reply TUTORIAL.md:2413]: Please update the PR, linking to this image 👍 (Literally use `src="https://user-images.githubusercontent.com/16427929/98684805-e759f200-2366-11eb-8dd5-6f283898ed6f.PNG"`)

Comment (brentguf) [reply TUTORIAL.md]: Just did! Had to post this first. I couldn't upload an image straight in the markdown editor.

Comment (brentguf) [reply TUTORIAL.md]: I would also change the alt attributes to empty strings, because this isn't very helpful.

File: TUTORIAL.md [hunk 2]
```diff
@@ -2430,12 +2430,14 @@ Now we're going to [create a Netlify account](https://app.netlify.com/signup) if
 
 Now just authorize Netlify to connect to your git hosting provider and find your repo. When the deploy settings come up you can leave everything as the defaults and click **Deploy site**.
 
-Netlify will start building your app (click the **Deploying your site** link to watch the logs) and it will say "Site is live", but nothing will work. Why? We haven't told it where to find our database yet.
```
Comment (brentguf) [inline TUTORIAL.md:2433]: I couldn't see this _Deploying your site_ link anywhere. Maybe I missed it, or it's gone?

Comment (cannikin) [reply TUTORIAL.md:2433]: They might have changed the behavior when you setup a new site, they've been updating their UI a bunch lately. If you went straight to deploy screen where the log is displayed then let's just remove that mention of clicking the link!

Comment (brentguf) [reply TUTORIAL.md:2433]: @cannikin I was wrong! I didn't look closely enough, so reverted it back to the original text.

**Pull request merged**